openreview/venue/process/sac_paper_assignment_pre_process.js

```diff
@@ -0,0 +1,21 @@
+async function process(client, edge, invitation) {
+    client.throwErrors = false
+  
+    const { groups } = await client.getGroups({ id: edge.domain })
+    const domain = groups[0]
+    const venueId = domain.id
+    const submissionName = domain.content.submission_name?.value
+    const sacsName = invitation.content.reviewers_name?.value
+
+    const { notes } = await client.getNotes({ id: edge.head })
+    const submission = notes[0]
+    const submissionGroupId = `${venueId}/${submissionName}${submission.number}`
+
+    if (!edge.ddate) {
```

Comment:
do we have this validation with the other groups?

Reply:
Yes, the [default preprocess](https://github.com/openreview/openreview-py/blob/master/openreview/venue/process/assignment_pre_process.js) checks this but also checks if there is a review/metareview posted or if the signatory group does not exist, but neither of those apply to SACs. 
Do you think this should be the same preprocess for all and we check if it's an SAC?